universal/src/yaml_config/yaml_config.cpp

```diff
@@ -128,6 +153,12 @@ YamlConfig YamlConfig::operator[](std::string_view key) const {
       // Strip substitutions off to disallow nested substitutions
       return YamlConfig{std::move(*env_value), {}, Mode::kSecure};
     }
+
+    auto file_value = GetFromFileByKey(key, yaml_);
```

Comment:
Looks good!

Please also update the docs and examples at

https://github.com/userver-framework/userver/blob/0d821cfd62360cd58cdff526dd418138ed342e9e/universal/include/userver/yaml_config/yaml_config.hpp#L31-L60

Reply:
Thanks for a review! 
I have added a bit of documentation, could you please look one more time?